.github/scripts/check_ocp_indent.py

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env python3.8
```

Comment:
Ideally I'd have a setup.cfg and setup.py to indicate that the version should be >= 3.8. I omitted those because I was being lazy and because I didn't need to specify any external dependencies. However, the way I did this here may leave someone confused in the future if they try to run against <3.8 and see runtime exceptions because of some of the features I used here.

I'll add a minimal setup.py before committing this to do it right.